pkg/virt-handler/cmd-client/client_test.go

```diff
@@ -115,35 +114,32 @@ var _ = Describe("Virt remote commands", func() {
 			sock, err := FindSocketOnHost(vmi)
 			Expect(err).ToNot(HaveOccurred())
 
+			Expect(IsSocketUnresponsive(sock)).To(BeFalse())
+
 			os.RemoveAll(sock)
 
 			// unresponsive is true when no socket file exists
-			unresponsive := IsSocketUnresponsive(sock)
-			Expect(unresponsive).To(BeTrue())
+			Expect(IsSocketUnresponsive(sock)).To(BeTrue())
 
 			// unresponsive is false when socket file exists
 			os.Mkdir(filepath.Dir(sock), 0755)
 			f, err := os.Create(sock)
 			Expect(err).ToNot(HaveOccurred())
 			f.Close()
```

Comment:
I think we have to ensure the socket was responsive before marking it unresponsive:
`Expect(IsSocketUnresponsive(sock)).To(BeFalse())`
Otherwise it can be true because it never becomes false

Reply:
Added.